clients/lightrider/lightrider-entry.js

```diff
@@ -25,9 +25,9 @@ const LR_PRESETS = {
  * If configOverride is provided, lrDevice and categoryIDs are ignored.
  * @param {Connection} connection
  * @param {string} url
- * @param {LH.Flags} flags Lighthouse flags, including `output`
+ * @param {LH.Flags} flags Lighthouse flags
  * @param {{lrDevice?: 'desktop'|'mobile', categoryIDs?: Array<string>, logAssets: boolean, configOverride?: LH.Config.Json}} lrOpts Options coming from Lightrider
- * @return {Promise<string|Array<string>|void>}
+ * @return {Promise<string|void>}
  */
 async function runLighthouseInLR(connection, url, flags, lrOpts) {
   const {lrDevice, categoryIDs, logAssets, configOverride} = lrOpts;
```

Comment:
I'm unclear how the default for the output flag is handled. Could we be explicit and add `flags.output = 'json'` here https://github.com/GoogleChrome/lighthouse/pull/9016/files#diff-872604c92afd40cde2ae727a77bae800R41 ?

Reply:
it can, but that's the default (at some point you have to trust the defaults, because there are a whole lot of them :). I meant my reminder mostly because I know something used to be requesting `html` output